WAGTAIL3UPGRADE.md

```diff
@@ -0,0 +1,30 @@
+# Wagtail 3.0 upgrade issues
```

Comment:
Just calling out this temporary doc for changes we will need to implement.

I think we will need someone from the original team to give some guidance on this.